lib/logger.js

```diff
@@ -96,7 +96,10 @@ logger.banner = function () {
 	var info = appc.pkginfo.package(module, 'version', 'about');
 	if (bannerEnabled) {
 		logger.log(info.about.name.cyan.bold + ', CLI version ' + info.version + (logger.activeSdk ? ', Titanium SDK version ' + logger.activeSdk : '') + '\n' + info.about.copyright);
-		logger.log('\n' + __('Please report bugs to %s', 'https://github.com/appcelerator/titanium_mobile/issues'.cyan) + '\n');
+		logger.log('\n' + __('- Please report bugs here: %s', 'https://github.com/tidev/titanium_mobile/issues'.cyan));
+		logger.log(__('- Remember! Titanium is a community supported open-source project and relies on donations for on-going development.'));
+		logger.log(__('- Support the project here: %s', 'https://liberapay.com/tidev'.cyan));
+		logger.log(__('- Join the TiDev Slack community here: %s \n', 'https://tislack.org'.cyan));
```

Comment:
Why are we putting the Slack URL in the CLI output? This banner output is already too big, too many lines. I would suggest just displaying a link to https://titaniumsdk.com. That's where the docs are. That's where we want to funnel people to go first for help.